score/README.md

```diff
@@ -1,5 +1,149 @@
+> For example: `poetry run python application.py etl-run ejscreen` would only run the ETL process for EJSCREEN data.
+
+#### Step 2: Calculate the Justice40 scores
+
+1. Call the `score-run` command using the application manager `application.py`
+   - With Docker: `docker run --rm -it j40_score /bin/sh -c "python3 application.py score-run"`
+   - With Poetry: `poetry run python application.py score-run`
+1. The `score-run` command will execute the `etl/score/etl.py` script which loads the data from each of the source files added to the `data/dataset/` directory by the ETL scripts in Step 1.
+1. These data sets are merged into a single dataframe using their Census Block Group GEOID as a common key, and the data in each of the columns is standardized in two ways:
+    - Their [percentile rank](https://en.wikipedia.org/wiki/Percentile_rank) is calculated, which tells us what percentage of other Census Block Groups have a lower value for that particular column.
+    - They are normalized using [min-max normalization](https://en.wikipedia.org/wiki/Feature_scaling), which adjusts the scale of the data so that the Census Block Group with the highest value for that column is set to 1, the Census Block Group with the lowest value is set to 0, and all of the other values are adjusted to fit within that range based on how close they were to the highest or lowest value.
+1. The standardized columns are then used to calculate each of the [Justice40 candidate scores](#justice40-candidate-scores) described in greater detail below, and the results are exported to a `.csv` file in [`data/score/csv`](data/score/csv)
+
+#### Step 3: Compare the Justice40 scores to other indices
+
+1. TODO: Describe the steps for this
+
+### Data Sources
+
+- **[EJSCREEN](etl/sources/ejscreen):** TODO Add description of data source
+- **[Census](etl/sources/census):** TODO Add description of data source
+- **[American Communities Survey](etl/sources/census_acs):** TODO Add description of data source
+- **[Housing and Transportation](etl/sources/housing_and_transportation):** TODO Add description of data source
+- **[HUD Housing](etl/sources/hud_housing):** TODO Add description of data source
+- **[HUD Recap](etl/sources/hud_recap):** TODO Add description of data source
+- **[CalEnviroScreen](etl/scores/calenviroscreen):** TODO Add description of data source
+
+### Justice40 candidate scores
+
+Below we've outlined some experimental versions of the Justice40 score that this comparative tool seeks to compare against the well-established environmental justice indices listed above.
+
+>**NOTE:** These scores _**do not**_ represent final versions of the Justice40 scores and are merely used for comparative purposes. As a result, the specific input columns and formulas used to calculate them are likely to change over time.
```

Comment:
Good idea to add this note! Let's make this just a normal paragraph (no block quotes) and move to line 40, right under `Score comparison workflow`